python_modules/dagster-airflow/setup.py

```diff
@@ -41,8 +41,6 @@ def _do_setup(name='dagster-airflow'):
             'enum34>=1.1.6',
             'future>=0.16.0',
             'apache-airflow==1.10.2',
-            'black==18.3a0 ; python_version>=\'3.6\'',
```

Comment:
thanks for accommodating this request. i know it hurt your soul a bit

Reply:
I did like using the indenting printer